src/web/client/extension.ts

```diff
@@ -92,7 +92,7 @@ export function activate(context: vscode.ExtensionContext): void {
                         );
                     }
                 }
-                const geo = queryParamsMap.get('geo')?.toLowerCase();
+                const geo = queryParamsMap.get(queryParameters.ORG_GEO)?.toLowerCase() ?? 'us';
```

Comment:
1. Given this field determines where the telemetry data will be logged, I don't think we should have a fallback option here.
2. We should make 'geo' as a mandatory parameter.
3. We should think if this is the correct way to solve the problem of determining the cluster for logging telemetry data. A user can easily modify the url parameter and change the value of ORG_GEO in which case we would end up logging data in an incorrect cluster. I think we should leverage a service from VSCode e.g., Artemis to determine this info.

cc: @tyaginidhi

Reply:
We already have Artemis integration for vscode.dev, we can leverage that here.

Reply:
@ashishchoudhary001 Called Artemis service to get the GeoName